packages/block-editor/src/components/rich-text/format-toolbar-container.js

```diff
@@ -19,6 +19,7 @@ const FormatToolbarContainer = ( { inline, anchorRef } ) => {
 				focusOnMount={ false }
 				anchorRef={ anchorRef }
 				className="block-editor-rich-text__inline-format-toolbar"
+				__unstableSlotName="block-toolbar"
```

Comment:
What does this do?

Reply:
It makes the format toolbar use a popover slot located inside the interface skeleton (same as the block toolbar uses), which is important because the `show-icon-labels` class is added to the skeleton container. By default the toolbar would use the popover slot outside it, so it wouldn't get the relevant styles.